dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/interruptedjobwindow/JobInfoListTreeTableItem.java

```diff
@@ -156,7 +156,7 @@ private void buildChildren(final ObservableList<TreeItem<JobInfoModel>> children
                         }
                     } else {
                         final GetObjectsWithFullDetailsSpectraS3Request request = new GetObjectsWithFullDetailsSpectraS3Request().withBucketId(modelValue.getBucket()).withIncludePhysicalPlacement(true);
-                        request.withName(modelValue.getName() + StringConstants.PERCENT).withFolder(modelValue.getName());
```

Comment:
Do we know that this is safe to remove with no change in behavior?

Reply:
We had removed ```withFolder()``` some time ago, but it seems to have come back. I worked with @hughartk, at the time to remove it. It seems to be the correct choice.

Reply:
Huh, I wonder how it came back. I wonder if it was something that was in my branch for whatever reason.